### PR TITLE
Fix bug with reconciling configmaps on deletion

### PR DESCRIPTION
**Pull request by rossf7**
For the values configmap we support managing configmaps in multiple resources. I also implemented deleting missing configmaps on update.

This causes chaos because it deletes the configmaps in kube-system created by k8scloudconfig 😱 

Since cluster-operator doesn't control the whole namespace it can't delete configmaps on update. They will be deleted correctly when the cluster is deleted.

### CODE REVIEW

File: service/controller/aws/v5/version_bundle.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ func VersionBundle() versionbundle.Bundle {
 		Changelogs: []versionbundle.Changelog{
 			{
 				Component:   "cluster-operator",
-				Description: "Add your changes here.",
+				Description: "Added configmap resource for managing chart configuration in guest clusters.",
```
Comment (rossf7) [inline service/controller/aws/v5/version_bundle.go]: Also spotted the change log was missing for AWS.

File: service/controller/kvm/v5/version_bundle.go [hunk 2]
```diff
@@ -9,7 +9,7 @@ func VersionBundle() versionbundle.Bundle {
 		Changelogs: []versionbundle.Changelog{
 			{
 				Component:   "cluster-operator",
-				Description: "Add your changes here.",
+				Description: "Added configmap resource for managing chart configuration in guest clusters.",
```
Comment (rossf7) [inline service/controller/kvm/v5/version_bundle.go]: Same for KVM.

**Comment by kopiczko** [comment]
For my interest, what will now delete those CMs?

**Comment by rossf7** [comment]
They will be deleted when the cluster is deleted.

**Comment by kopiczko** [comment]
I needed time to digest that, but I understand now. This is for ApplyUpdate :)

**Pull request merged**